components/authz-service/storage/postgres/migration/sql/60_use_db_id_as_primary_key_in_iam_statement_projects.up.sql

```diff
@@ -0,0 +1,165 @@
+BEGIN;
+
+ALTER TABLE iam_statement_projects
+    DROP CONSTRAINT iam_statement_projects_project_id_fkey;
```

Comment:
Wait, is that the right constraint to drop? The `project_id` part of this isn't altered, but the `statement_id` part -- or am I mistaken?

Reply:
Ah yeah that's incorrect. I'll fix that up.